jekyll/_cci2_ja/server-3-operator-proxy.adoc

```diff
@@ -18,15 +18,15 @@ toc::[]
 プロキシ経由で CircleCI サーバーをインストールするには、2つの手順があります。 まずインストールをする際に、プロキシアドレスとプロキシの後ろにあってはいけないアドレスを指定する必要があります。
 
 === プロキシ経由でのインストール
-設定方法の詳細は、https://circleci.com/docs/2.0/server-3-install/[CircleCI Server v3.x インストールガイド]を参照してください。 プロキシアドレスと非プロキシアドレスの両方に、 https://kots.io/kotsadm/installing/online-install/#proxies[こちら]で説明されている引数を使用する必要があります。 インストールコマンドは、次のような形式で指定します。
+設定方法の詳細は、 https://circleci.com/docs/2.0/server-3-install/[CircleCI Server v3.x インストールガイド] を参照してください。 プロキシアドレスと非プロキシアドレスの両方に、 https://kots.io/kotsadm/installing/online-install/#proxies[こちら]で説明されている引数を使用する必要があります。 インストールコマンドは、次のような形式で指定します。
 
 [source,bash]
 ----
 kubectl kots install circleci-server --http-proxy <my-http-proxy-uri> --https-proxy <my-https-proxy> --no-proxy <my-no-proxy-list>
 ----
 
 === プロキシの設定
-サーバーをインストールし、管理コンソールにアクセスできるようになったら、以下のスクリーンショットで示すように、設定セクションで入力しなければならない項目があります。 これらのフィールドは自動的には入力されないので、インストール時に入力したプロキシアドレスと非プロキシアドレスをご自身で入力する必要があります。 プロキシがユーザー名とパスワードによる認証を必要とする場合は、*HTTP Proxy authenticated(HTTP プロキシ認証済)*オプションにチェックを入れ、認証情報を追加します。
+サーバーをインストールし、管理コンソールにアクセスできるようになったら、以下のスクリーンショットで示すように、設定セクションで入力しなければならない項目があります。 これらのフィールドは自動的には入力されないので、インストール時に入力したプロキシアドレスと非プロキシアドレスをご自身で入力する必要があります。 プロキシがユーザー名とパスワードによる認証を必要とする場合は、 *HTTP Proxy authenticated(HTTP プロキシ認証済)* オプションにチェックを入れ、認証情報を追加します。
```

Comment:
I would recommend to fix the following to `**HTTP Proxy authenticated(HTTP プロキシ認証済)**` so that it would be bold.

https://circleci.com/docs/2.0/server-3-operator-proxy/#configuring-your-proxy

<img width="756" alt="image" src="https://user-images.githubusercontent.com/6015450/169994274-eac48d70-cf66-4531-90f5-0b82428b72eb.png">